cms/templates/js/add-xblock-component-menu.underscore

```diff
@@ -1,23 +1,29 @@
 <% if (type === 'advanced' || templates.length > 1) { %>
-    <div class="tab current" id="tab1">
+    <div class="tab current" id="tab1" tabindex="-1" role="group" aria-label="<%-
```

Comment:
Since the new menu that appears has choices, it makes more sense to have this appear like a dialog box.